tests/render/test_util.py

```diff
@@ -255,15 +255,15 @@ def load_notebook_from_path(
 def find_code_in_notebook(
     nb: NotebookNode,
     search_string: str,
-) -> Optional[Dict[int, dict]]:
```

Comment:
@cdkini I believe that this change is incorrect.  The callers to this `find_code_in_notebook()` rely on the `None` return.  Thanks.

Reply:
I made this change since we do some `len` assertions against the resulting object (which resulted in unhelpful errors due to it being a `NoneType`).

Very minor and you have more context here so I'll revert the changes.

Reply:
🙇 -- thank you!  Makes me feel safer about the very tedious and painful to troubleshoot tests involving Jupyter notebooks.